erpnext_autoinstall/install.py

```diff
@@ -0,0 +1,78 @@
+import os
+
+import frappe
+
+
+def after_install():
+    disable_registration()
+    configure_email()
+    configure_ldap()
+
+
+def disable_registration():
+    doc = frappe.get_doc("Website Settings")
+    doc.disable_signup = int(os.getenv('DISABLE_SIGNUP', '0'))
+    doc.save()
+
+
+def configure_domain():
+    if os.getenv('EMAIL SERVER') and os.getenv('EMAIL_ADDRESS') and os.getenv('EMAIL_DOMAIN_NAME') and os.getenv(
+            'EMAIL_SMTP_SERVER') and os.getenv('EMAIL_PORT') and os.getenv('EMAIL_USE_IMAP') and os.getenv(
+        'EMAIL_USE_SSL') and int(os.getenv('EMAIL_USE_TLS')) and os.getenv('EMAIL_ATTACHMENT_LIMIT_MB'):
```

Comment:
@AminovE99 We should probably add default values to the most common email properties (port 993 and use SSL true for instance)

Reply:
Done